test/commands/org/logout.test.ts

```diff
@@ -151,11 +151,11 @@ describe('org:logout', () => {
     expect(response).to.deep.equal([testOrg1.username]);
   });
 
-  it('should not fail when the auth file does not exist', async () => {
+  it('should fail when the auth file does not exist', async () => {
     await prepareStubs({
       'target-org': testOrg2.username,
       aliases: { TestAlias: testOrg1.username },
-      authInfoConfigDoesNotExist: true,
+      authInfoConfigDoesNotExist: false,
```

Comment:
this option's name is very confusing, setting it to `false` means it exists. we should be asserting that somewhere an error is thrown.

I would've expected some of the assertions to change if we changed the behavior of the UT